CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+# How to Contribute to the OpenTelemetry Operator
+
+We'd love your help!
+
+This project is [Apache 2.0 licensed](LICENSE) and accepts contributions via GitHub pull requests. This document outlines some of the conventions on development workflow, contact points and other resources to make it easier to get your contribution accepted.
+
+We gratefully welcome improvements to documentation as well as to code.
+
+## Getting Started
+
+### Workflow
+
+It is recommended to follow the ["GitHub Workflow"](https://guides.github.com/introduction/flow/). When using [GitHub's CLI](https://github.com/cli/cli), here's how it typically looks like:
+
+```
+$ gh repo fork github.com/open-telemetry/opentelemetry-operator
+$ git checkout -b your-feature-branch
+# do your changes
+$ git commit -sam "Add feature X"
+$ gh pr create
+```
+
+### Pre-requisites
+* Install [Go](https://golang.org/doc/install).
+* Have a Kubernetes cluster ready for development. We recommend `minikube` or `kind`.
+
+### Local run
+
+Build the manifests, install the CRD and run the operator as a local process:
+```
+$ make manifests install run
+```
+
+### Deployment with webhooks
+
+When running `make run`, the webhooks aren't effective as it starts the manager in the local machine instead of in-cluster. To test the webhooks, you'll need to:
+
+1. configure a proxy between the Kubernetes API server and your host, so that it can contact the webhook in your local machine
+1. create the TLS certificates and place them, by default, on `/tmp/k8s-webhook-server/serving-certs/tls.crt`. The Kubernetes API server has also to be configured to trust the CA used to generate those certs.
+
+In general, it's just easier to deploy the manager in a Kubernetes cluster instead. For that, you'll need the `cert-manager` installed. You can install it by running:
+
+```console
+make cert-manager
+```
+
+Once it's ready, the following can be used to build and deploy a manager, along with the required webhook configuration:
+
+```
+make manifests docker-build docker-push deploy
+```
+
+By default, it will generate an image following the format `quay.io/${USER}/opentelemetry-operator:${VERSION}`. You can set the following env vars in front of the `make` command to override parts or the entirety of the image:
+
+* `USER`, to override the namespace
+* `VERSION`, to override only the version part
+* `IMG_PREFIX`, to override the registry, namespace and image name (`quay.io/${USER}/opentelemetry-operator`)
```

Comment:
Not necessarily a docs issue, but this doesn't seem to be a prefix, when it represents a majority of the image specification?

Reply:
Something like this?

```
IMG_PREFIX = "quay.io"
IMG_NAMESPACE = "${USER}"
IMG_REPO = "opentelemetry-operator"
IMG = "${IMG_PREFIX}/${IMG_NAMESPACE}/${IMG_REPO}:${VERSION}"
```

Reply:
Yeah better +1

Reply:
Done, but left the IMG_NAMESPACE out, as it can be overridden already